src/test/v2.0/index.ts

```diff
@@ -73,6 +72,31 @@ function toggleBackButton() {
   }
 }
 
+function executeSingleTest(testName: string, manualTest: boolean) {
```

Comment:
this looks like it's repeated code

Reply:
I can sort it out and make it as one function.